doc/setup_instructions.md

```diff
@@ -1,6 +1,5 @@
 What's this?
 ============
-
```

Comment:
Please don't remove this blank line. All the other titles have a following blank line